src/ui/ReferenceView.cpp

```diff
@@ -534,10 +534,13 @@ void ReferenceView::contextMenuEvent(QContextMenuEvent *event)
   }
 
   if (ref.isRemoteBranch()) {
-    menu.addAction(tr("New Local Branch"), [this, ref] {
-      QString local = ref.name().section('/', 1);
+    QString local = ref.name().section('/', 1);
+    QAction *newLocalBranch = menu.addAction(tr("New Local Branch %1").arg(local),
```

Comment:
Don't you have to update a bunch of translation files due to the added %1?

Reply:
Yes, it will, but I think this is probably the best way to make it clear (rather than hiding the menu itself, or just graying it out).

Reply:
Sorry, that's not what I meant.

Shouldn't you touch/change [this line](https://github.com/gitahead/gitahead/blob/ca367feacde0fd23af2d72e93b2f3ec800ef6556/l10n/gitahead_ja.ts#L3041) and the corresponding lines in all the other translation files in the same commit?

Reply:
It's up to hackhaslam's decision when to rebase those translation files.